src/wp-includes/functions.wp-scripts.php

```diff
@@ -340,23 +345,26 @@ function wp_deregister_script( $handle ) {
  *                                    as a query string for cache busting purposes. If version is set to false, a version
  *                                    number is automatically added equal to current installed WordPress version.
  *                                    If set to null, no version is added.
- * @param bool             $in_footer Optional. Whether to enqueue the script before `</body>` instead of in the `<head>`.
- *                                    Default 'false'.
+ * @param array            $args      {
+ *      Optional. An array of additional script loading strategies. Default empty array.
+ *
+ *      @type boolean   $in_footer    Optional. Default false.
+ *      @type string    $strategy     Optional. Values blocking|defer|async. Default 'blocking'.
+ * }
  */
-function wp_enqueue_script( $handle, $src = '', $deps = array(), $ver = false, $in_footer = false ) {
+function wp_enqueue_script( $handle, $src = '', $deps = array(), $ver = false, $args = array() ) {
```

Comment:
```suggestion
function wp_enqueue_script( $handle, $src = '', $deps = array(), $ver = false, $args = array( 'in_footer' => false ) ) {
```
Does this make sense?

Reply:
Defaults for arrays are typically filled within the function logic, not the signature, in WP core, so I think we should stick with just the empty array here.